### ISSUE TITLE
widget

### ISSUE THREAD

**Issue by gargamel17** [bmaroti9/Overmorrow#11]
Thank you for the app 👍
Is it planned to add a widget ?

**Comment by bmaroti9** [comment]
Yes it is planned. I was looking through the tutorials on how to make them but i don't really understand 😥. But i will try another session where i just go through them and try to learn it.

**Comment by BloodRaven0** [comment]
@bmaroti9 You can do it, we believe in you! ❤️

**Comment by bmaroti9** [comment]
Hmm, that's very strange. I have no idea what's causing it. Does it also happen with the other widgets? Without proper debugging I can't know what's happening. I plan on adding specifically more widget debugging tools either as a section in the app, or as a separate widget. Untill then all I can think of are basic stuff like checking if your connected to wifi, or trying to delete and reinstall.

**Comment by GabboRocchia** [comment]
I tried reinstalling it and it worked but the problem wasn't just that.
The first time I installed it I placed the widget before opening the app and giving the location permission (I just choose the default location Nashville). Then I opened the app, gave all the permissions and loaded the weather info of my current location. After that I went back to the widget and modified the location to my current one and that didn't worked giving the error above.
After reinstalling, opening the app and giving location permissions before placing the widget everything worked fine.
I tried to reinstall and make the same step as the first time and I got the same problem so it should be reproducible.
Not really sure if the problem was the permission or the fact that I didn't open the app but I think it's worth to signal.
Thank you very much for the great app!
